src/booleans.js

```diff
@@ -1,62 +1,68 @@
 function negate(a) {
-  // your code here
+// NOTE GATE
+  return (!a);
 };
 
 function both(a, b) {
-  // your code here
+// AND GATE
+  return (a&&b);
```

Comment:
Dont need the parenthesis here, and you need to add some spaces for better formatting.
`````
return a && b
``````